packages/frontend/src/features/sqlRunner/hooks/useSqlQueryRun.tsx

```diff
@@ -0,0 +1,171 @@
+import {
+    SchedulerJobStatus,
+    type ApiError,
+    type ApiJobScheduledResponse,
+    type ApiJobStatusResponse,
+    type ResultRow,
+    type SqlRunnerBody,
+} from '@lightdash/common';
+import { useMutation, useQuery, useQueryClient } from '@tanstack/react-query';
+import { useMemo } from 'react';
+import { lightdashApi } from '../../../api';
+import useToaster from '../../../hooks/toaster/useToaster';
+import { getSchedulerJobStatus } from '../../scheduler/hooks/useScheduler';
+import { useAppSelector } from '../store/hooks';
+
+const scheduleSqlJob = async ({
+    projectUuid,
+    sql,
+}: {
+    projectUuid: string;
+    sql: SqlRunnerBody['sql'];
+}) =>
+    lightdashApi<ApiJobScheduledResponse['results']>({
+        url: `/projects/${projectUuid}/sqlRunner/run`,
+        method: 'POST',
+        body: JSON.stringify({ sql }),
+    });
+
+/**
+ * Gets the SQL query results from the server
+ *
+ * Steps:
+ * 1. Schedule the SQL query job
+ * 2. Get the status of the scheduled job
+ * 3. Fetch the results of the job
+ */
+export const useSqlQueryRun = () => {
+    const { showToastError } = useToaster();
+    const projectUuid = useAppSelector((state) => state.sqlRunner.projectUuid);
+    const queryClient = useQueryClient();
+    const sqlQueryRunMutation = useMutation<
+        ApiJobScheduledResponse['results'],
+        ApiError,
+        {
+            projectUuid: string;
```

Comment:
We can remove this param.

Reply:
oh yes! 
[chore: remove unused mutation param](https://github.com/lightdash/lightdash/pull/10608/commits/625fae41c50b55454b13d3bb717fe9a58882ff87)